lib/tasks/ci.rake

```diff
@@ -9,6 +9,9 @@ begin
 
   namespace :ci do
     task :all do
+      # Run rubocop on the current branches commit diff with master:
+      # rubocop:disable Metrics/LineLength
+      # system!("git diff-tree -r --no-commit-id --name-only master@\{u\} head | xargs ls -1 2>/dev/null | grep '\.rb$' | xargs rubocop")
```

Comment:
This can be used for existing projects where it would be too much effort to fix all the rubocop issues right away. This will only check the files that you have changed in your git diff.